pkgs/development/python-modules/uamqp/default.nix

```diff
@@ -18,14 +22,25 @@ buildPythonPackage rec {
     sha256 = "sha256-zDUFe/yMCThn+qJqDekMrUHEf1glGxBw4pioExLLoqg=";
   };
 
+  # can be removed when https://github.com/Azure/azure-c-shared-utility/pull/499 is in the vendored files
```

Comment:
fetchpatch does not work here?

Reply:
I did the PR afterwards. The files are vendored and the last time I fiddled with patching vendors it took me way to long to get it working. I guess we could do it here but I am lazy right now.

Reply:
Guess I got better and fixed it right away.